core/src/main/java/org/ironjacamar/core/connectionmanager/pool/dflt/DefaultManagedConnectionPool.java

```diff
@@ -426,8 +426,8 @@ public void increaseCapacity()
                boolean added = false;
                if (listeners.size() < pool.getConfiguration().getMaxSize())
                {
-                              /*if (trace)
-                                 log.trace("Capacity fill: cl=" + cl);*/
+                  if (log.isTraceEnabled())
+                     log.trace("Capacity fill: cl=" + cl);
```

Comment:
Messages at DEBUG and TRACE level don't need translation, so it's fine to leave it inline in the code